### PR TITLE
Added the New Relic Metrics Adapter

### PR DESCRIPTION
**Pull request by mangulonr**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
From New Relic we want to add our external metrics adapter to the implementations list.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
- [Usage]: https://github.com/kuperiu/k8s-newrelic-adapter
- [Other doc]: https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/newrelic-hpa-metrics-adapter/newrelic-metrics-adapter/
```
/check-cla

### CODE REVIEW

File: staging/src/k8s.io/metrics/IMPLEMENTATIONS.md [hunk 1]
```diff
@@ -39,3 +39,5 @@ They are listed here for convenience.***
   They are listed here for convenience.***
 
  - [KEDA](https://github.com/kedacore/keda). A Kubernetes-based Event Driven Autoscaling component. It provides event driven scale for any container running in Kubernetes.
+
+- [New Relic Metrics Adapter](https://github.com/newrelic/newrelic-k8s-metrics-adapter). Implementation of the external metrics provider. It provides the capability to autoscale the cluster using New Relic telemetry data like infrastucture services or applications. More information [here](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/newrelic-hpa-metrics-adapter/newrelic-metrics-adapter/).
```
Comment (roobre) [inline staging/src/k8s.io/metrics/IMPLEMENTATIONS.md:43]:
```suggestion
- [New Relic Metrics Adapter](https://github.com/newrelic/newrelic-k8s-metrics-adapter). Exposes New Relic metrics and query results as External Metrics, allowing to scale your workloads using data from infrastructure, APM, or a combination of both. More information [here](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/newrelic-hpa-metrics-adapter/newrelic-metrics-adapter/).
```

**Comment by ehashman** [comment]
Thanks for raising this. It seems you don't have a CLA signed so it's not possible to merge this PR.

We discussed at the SIG Instrumentation meeting today and we are planning on removing references to external implementations, not supported by Kubernetes, as most of this information is extremely out of date.

File: staging/src/k8s.io/metrics/IMPLEMENTATIONS.md (hunk 1 above)
Comment (mangulonr) [reply staging/src/k8s.io/metrics/IMPLEMENTATIONS.md:43]: I signed it

**Comment by mangulonr** [comment]
I signed it

**Comment by roobre** [comment]
Hi @ehashman,
Thanks for the information on the upcoming removal of this list. Do you happen to know if this removal is taking effect sometime soon? For us it was a nice point to get this new feature showcased here, but if it's going away very soon we might stop fighting our bureaucracy to get the CLA signed :)
Thanks!

**Comment by dims** [comment]
Is this PR still needed, please rebase if so (or we can close it?)

**Comment by dims** [comment]
This PR needs a CLA to make progress, please see comments above from the CLA bot(s)

**Comment by dims** [comment]
There is a cncf-cla:no label on this PR. Can you please see the bot comments above to fix the CLA? This PR cannot progress unless the bot is happy with the CLA situation

**Comment by roobre** [comment]
@mangulonr Could you check the CLA issue above? Thanks!

**Comment by dims** [comment]
@sftim Do we want to move this sort of stuff to docs?

**Review by sftim — COMMENTED** [review]
In my view, a better change given https://kubernetes.io/blog/2020/05/third-party-dual-sourced-content/ and the related KEP is to remove the external implementations list from this repo, and replace it with a new list or table within https://k8s.io/docs/reference/

The new list should **not**:
- make statements about the future (“coming soon”, etc)
- appraise or outline features of the implementations
   - all I'd expect to see is the name of the implementation, which would hyperlink to a page about it

The new list should be in alphabetical order and add the usual disclaimers. It's OK to replace this existing file with a Markdown stub that links to the new page.

**Comment by sftim** [comment]
If someone's willing to open an issue about doing that migration, I'd appreciate it!

**Pull request closed without merging**